packages/@okta/vuepress-site/docs/reference/hook-best-practices/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hook best practices
+excerpt: >-
+  Understand the best practices at Okta to implement and secure an Okta Event Hook or Inline Hook.
+---
+
+# Hook best practices
+
+Event Hooks and Inline Hooks are outbound calls from Okta to an external service. These outbound calls integrate custom functionality into your Okta processes. For further background information on Okta Hooks, see [Event Hooks](/docs/concepts/event-hooks) and [Inline Hooks](/docs/concepts/inline-hooks).
```

Comment:
See [Event Hooks](/docs/concepts/event-hooks) and [Inline Hooks](/docs/concepts/inline-hooks) for additional concept information.
(following the style guide to put the "See" first)